packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -194,9 +194,15 @@ const dirName = path => {
  * files associated to a specific document
  */
 class FileCollection extends DocumentCollection {
-  constructor(doctype, stackClient) {
-    super(doctype, stackClient)
+  constructor(doctype, stackClient, options = {}) {
```

Comment:
nit: Maybe we can type with TS comment this option

Reply:
Yes, agreed! I did the minimum I had to to test it and show it to you guys. I plan on adding tests (and types here then).